shunting-yard-common/src/main/java/com/expediagroup/shuntingyard/common/receiver/thrift/ExpressionBuilder.java

```diff
@@ -1,10 +1,5 @@
 /**
```

Comment:
Can you check whether the code in `ExpressionBuilder` does look like it came from this Hive class mentioned below? Or maybe the javadoc was copied over incorrectly? If not and the code here is partly from the Hive project then this javadoc needs to stay and we need a section in the pom configuring the license plugin telling it not to overwrite this file (see https://github.com/HotelsDotCom/circus-train/blob/master/pom.xml#L283 for an example).

Reply:
😮 Nice Catch !

Actually, It was excluded in the pom xml. Because the packages got renamed, the class didn't get excluded. I have fixed it.